src/WP_CLI/CommandWithMeta.php

```diff
@@ -142,6 +142,15 @@ function ( $a, $b ) use ( $orderby, $order ) {
 	 * <key>
 	 * : The name of the meta field to get.
 	 *
+	 * [--is-single=<is-single>]
+	 * : Whether to return a single value. This parameter has no effect if $key is not specified.
+	 * ---
+	 * default: 1
+	 * options:
+	 *   - 1
+	 *   - 0
+	 * ---
```

Comment:
We don't typically use such a format in WP-CLI for boolean flags. Instead we use just `--is-single` for "true", and don't pass a flag if you want "false" (or `--no-is-single`, which is automatically supported)

So this would be simply:

```
* [--is-single]
* : Whether to return a single value. This parameter has no effect if $key is not specified.
```

And then you can use `$is_single = Utils\get_flag_value( $assoc_args, 'is-single', true );`

Also, I'd probably rename to just `--single`

Reply:
Got it. Making changes. Thank you for your feedback.